frontend/pages/editions.tsx

```diff
@@ -78,11 +81,12 @@ const Editions: NextPage = () => {
 
   const updateEdition = (_edition: string) => {
     setEdition(_edition);
-    router.push(''); // TODO. update this when students/projects page has been added
-    console.log(_edition);
+    router.push(`/${_edition}/projects`);
   };
 
   const deleteEdition = async (_edition: string) => {
+    if (!_edition) return;
```

Comment:
Why is this check required? Surely this function should never be called with the empty string as argument right?

Reply:
This is an extra safeguard. Saying something shouldn't happen, doesn't imply it will never happen.